gtfs/spec/en/reference.md

```diff
@@ -140,19 +142,19 @@ File: **Required**
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
 |  `stop_id` | ID | **Required** | Identifies a location: stop/platform, station, entrance/exit, generic node or boarding area (see `location_type`). <br><br>Multiple routes may use the same `stop_id`. |
-|  `stop_code` | Text | Optional | Short text or a number that identifies the location for riders. These codes are often used in phone-based transit information systems or printed on signage to make it easier for riders to get information for a particular location. The `stop_code` can be the same as `stop_id` if it is public facing. This field should be left empty for locations without a code presented to riders. |
-|  `stop_name` | Text | **Conditionally Required** | Name of the location. Use a name that people will understand in the local and tourist vernacular.<br><br>When the location is a boarding area (`location_type=4`), the `stop_name` should contains the name of the boarding area as displayed by the agency. It could be just one letter (like on some European intercity railway stations), or text like “Wheelchair boarding area” (NYC’s Subway) or “Head of short trains” (Paris’ RER).<br><br>Conditionally Required:<br>- **Required** for locations which are stops (`location_type=0`), stations (`location_type=1`) or entrances/exits (`location_type=2`).<br>- Optional for locations which are generic nodes (`location_type=3`) or boarding areas (`location_type=4`).|
+|  `stop_code` | Text | Optional | Short text or a number that identifies the location for riders. These codes are often used in phone-based transit information systems or printed on signage to make it easier for riders to get information for a particular location. The `stop_code` may be the same as `stop_id` if it is public facing. This field should be left empty for locations without a code presented to riders. |
+|  `stop_name` | Text | **Conditionally Required** | Name of the location. Names should be meaningful to local and tourist vernacular.<br><br>When the location is a boarding area (`location_type=4`), the `stop_name` should contains the name of the boarding area as displayed by the agency. It could be just one letter (like on some European intercity railway stations), or text like “Wheelchair boarding area” (NYC’s Subway) or “Head of short trains” (Paris’ RER).<br><br>Conditionally Required:<br>- **Required** for locations which are stops (`location_type=0`), stations (`location_type=1`) or entrances/exits (`location_type=2`).<br>- Optional for locations which are generic nodes (`location_type=3`) or boarding areas (`location_type=4`).|
```

Comment:
"meaningful to ... vernacular" does not seem like idiomatic English, so cannot be substituted for "in the ... vernacular". The expression "tourist vernacular" is a little hard to interpret anyway in the original spec. A possible alternative is "Names should be understandable to both local residents and tourists".

Reply:
I chose the word "meaningful" as a substitute for "will understand" to indicate that names should represent stops according to how people ("locals" and "tourists") associate names with place. That is to say, not only should the name be legible, but it should reflect what the stop is conventionally called. 

The suggested alternative to "Names should be understandable to both local residents and tourists" might be too ambiguous IMHO, as it may only allude to legibility and localization for different audiences (but say nothing about producing "meaningful" names).

@flocsy also [raised the point](https://github.com/google/transit/pull/277#discussion_r646094801) that we should add a note about the use of `translations.txt` along the lines of "local and tourist" (perhaps a separate issue).

Reply:
I can see why you might want to replace "understand" with some other word, but my sense is that the revised version of the sentence is not grammatically correct, specifically the expression "meaningful to ... vernacular". Perhaps the whole sentence should be rewritten once a clearer meaning is agreed upon, and perhaps without using the word "vernacular". As mentioned in other comments below about time fields, this change seems out of scope in a PR described as being about RFC 2119 terms. I'm not insisting on PRs being purely single-subject, but if a PR does include additional changes it seems important to mention those in the description or title.

Reply:
I agree that this needs more attention, and that it is a separate topic from RFC. I reverted it to the original for now.